tasks/docker.rb

```diff
@@ -149,8 +146,12 @@ def provision(image, inventory_location, vars)
   unless vars.nil?
     var_hash = YAML.safe_load(vars)
     node['vars'] = var_hash
+    docker_run_opts = var_hash['docker_run_opts'].flatten.join(' ') unless var_hash['docker_run_opts'].nil?
   end
-  group_name = 'ssh_nodes'
+  creation_command = "docker run -d -it --privileged #{deb_family_systemd_volume} --tmpfs /tmp:exec -p #{front_facing_port}:22 --name #{full_container_name} #{docker_run_opts} #{image}"
```

Comment:
I think referencing `docker_run_opts` if its nil, will cause an exception here. I would perhaps build up the `creation_command` like this:
```ruby
creation_command = "docker run -d -it --privileged #{deb_family_systemd_volume} --tmpfs /tmp:exec -p #{front_facing_port}:22 --name #{full_container_name} "
creation_command += "#{docker_run_opts} " unless var_hash['docker_run_opts'].nil?
creation_command += image
```

Reply:
Sounds good to me. This has been fixed and it works with `vars` specified and absent.